<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1541341890766" stop="1541341911537">
	<name>Test Gmail</name>
	<title>Test Gmail</title>
	<test-cases>
		<test-case start="1541341890767" stop="1541341895819" status="failed">
			<name>Validate user credentials</name>
			<title>Validate user credentials</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="e389428b-2758-4878-b7a4-5ac48bc35d75-attachment.png" type="image/png" size="35915"/>
			</attachments>
			<failure>
				<message>Failed: No element found using locator: by.buttonText(&quot;Next&quot;)</message>
				<stack-trace>NoSuchElementError: No element found using locator: by.buttonText(&quot;Next&quot;)
    at elementArrayFinder.getWebElements.then (/usr/local/lib/node_modules/protractor/built/element.js:814:27)
    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
    at ElementFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
    at UserContext.&lt;anonymous&gt; (/home/ketan/WebstormProjects/TestSuites/GmailTest.js:36:44)
    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
From: Task: Run it(&quot;Validate user credentials&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4264:13
    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4172:9
    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:64:48
    at ControlFlow.emit (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/events.js:62:21)
    at ControlFlow.shutdown_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2674:10)
    at shutdownTask_.MicroTask (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2599:53)
From asynchronous test: 
Error
    at /home/ketan/WebstormProjects/TestSuites/GmailTest.js:33:9
    at /home/ketan/WebstormProjects/TestSuites/node_modules/jasmine-data-provider/src/index.js:25:22
    at Array.forEach (&lt;anonymous&gt;)
    at /home/ketan/WebstormProjects/TestSuites/node_modules/jasmine-data-provider/src/index.js:20:20
    at Suite.&lt;anonymous&gt; (/home/ketan/WebstormProjects/TestSuites/GmailTest.js:31:5)
    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (/home/ketan/WebstormProjects/TestSuites/GmailTest.js:8:1)</stack-trace>
			</failure>
		</test-case>
		<test-case start="1541341895819" stop="1541341911536" status="failed">
			<name>Validate user credentials</name>
			<title>Validate user credentials</title>
			<description></description>
			<labels>
			</labels>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="507f949b-acae-4359-882c-8ce2c8b997aa-attachment.png" type="image/png" size="628630"/>
			</attachments>
			<failure>
				<message>Failed: No element found using locator: By(css selector, *[id=&quot;identifierId&quot;])</message>
				<stack-trace>NoSuchElementError: No element found using locator: By(css selector, *[id=&quot;identifierId&quot;])
    at elementArrayFinder.getWebElements.then (/usr/local/lib/node_modules/protractor/built/element.js:814:27)
    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)Error
    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
    at ElementArrayFinder.(anonymous function).args [as sendKeys] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
    at ElementFinder.(anonymous function).args [as sendKeys] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
    at UserContext.&lt;anonymous&gt; (/home/ketan/WebstormProjects/TestSuites/GmailTest.js:35:65)
    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:112:25
    at new ManagedPromise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)
    at ControlFlow.promise (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)
    at schedulerExecute (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:95:18)
    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
From: Task: Run it(&quot;Validate user credentials&quot;) in control flow
    at UserContext.&lt;anonymous&gt; (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4264:13
    at /usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4172:9
    at /usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:64:48
    at ControlFlow.emit (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/events.js:62:21)
    at ControlFlow.shutdown_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2674:10)
    at shutdownTask_.MicroTask (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2599:53)
From asynchronous test: 
Error
    at /home/ketan/WebstormProjects/TestSuites/GmailTest.js:33:9
    at /home/ketan/WebstormProjects/TestSuites/node_modules/jasmine-data-provider/src/index.js:25:22
    at Array.forEach (&lt;anonymous&gt;)
    at /home/ketan/WebstormProjects/TestSuites/node_modules/jasmine-data-provider/src/index.js:20:20
    at Suite.&lt;anonymous&gt; (/home/ketan/WebstormProjects/TestSuites/GmailTest.js:31:5)
    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
    at Object.&lt;anonymous&gt; (/home/ketan/WebstormProjects/TestSuites/GmailTest.js:8:1)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>